### PR TITLE
Add link to useful Turing Way chapters in the dev README guidelines

### PR DESCRIPTION
**Pull request by KirstieJane**
Made by Kirstie but with Rehaan and Paul during the March 11 AutSPACEs meetup.

### CODE REVIEW

File: dev_readme.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 #### 1. Download and install [Docker](https://www.docker.com/products/docker-desktop) for your operating system.
 
+You might find the chapter on [reproducible computational environments](https://the-turing-way.netlify.app/reproducible-research/renv.html) and spectifically the section on [containers](https://the-turing-way.netlify.app/reproducible-research/renv/renv-containers.html) in _The Turing Way_ useful!
```
Comment (KirstieJane) [inline dev_readme.md:5]: This is demonstrating a review! 🌟

**Review by KirstieJane — COMMENTED** [review]
This is Kirstie reviewing her own pull request to show @pgc007 how to review a PR 🚀

File: dev_readme.md (hunk 1 above)
Comment (jhlink) [reply dev_readme.md:5]: Can this be hyperlinked to a subsection within the `Extra Tips` section? 

I'd like to keep the instructions minimalistic.

Comment (jhlink) [reply dev_readme.md:5]: Can this be hyperlinked to a subsection within the `Extra Tips` section? 

I'd like to keep the instructions minimalistic.

**Review by jhlink — CHANGES_REQUESTED** [review]
Very nice add! Wondering if this could be move towards the end.

**Comment by GeorgiaHCA** [comment]
Hey @jhlink, Paul and Rehaan -  I know this was a test for demonstrating reviwing PRs, but I also think this would be a really useful thing to continue to work on? What do you reckon? We could learn together and make the repository more inclusive at the same time.

**Comment by jhlink** [comment]
The branch is on Kirstie's repo. I'd have to submit a PR to the `main` branch on Kirstie's repo to make any changes. 
This is fine, but it depends on Kirstie's availability. 

My suggestion is if the change is urgent enough, for the person who wants to push the PR, copy the change here and create a separate PR.

**Pull request merged**